content/en/docs/concepts/configuration/secret.md

```diff
@@ -238,123 +199,6 @@ You can use the [`kubectl create token`](/docs/reference/generated/kubectl/kubec
 command to obtain a token from the `TokenRequest` API.
 {{< /note >}}
 
-#### Projection of Secret keys to specific paths
-
-You can also control the paths within the volume where Secret keys are projected.
-You can use the `.spec.volumes[].secret.items` field to change the target path of each key:
-
-```yaml
-apiVersion: v1
-kind: Pod
-metadata:
-  name: mypod
-spec:
-  containers:
-  - name: mypod
-    image: redis
-    volumeMounts:
-    - name: foo
-      mountPath: "/etc/foo"
-      readOnly: true
-  volumes:
-  - name: foo
-    secret:
-      secretName: mysecret
-      items:
-      - key: username
-        path: my-group/my-username
-```
```

Comment:
what's the reason behind this migration?

Reply:
It's task style content in the concept page that was making the page bigger and also fragmenting secret-related instructions. Since we have a task page for using secrets in various ways already we decided to consolidate steps there and get rid of duplication